### PR TITLE
Default the RJ viewmodel to the centered version

### PR DESCRIPTION
**Pull request by Gocnak**
To match the default firing position cvar value.

### Checklist
- [x] **I have thoroughly tested all of the code I have modified/added/removed to ensure something else did not break**
- [x] If there is a localization token change, I have updated the `momentum_english_ref.res` file with the changes, ran `tokenizer.py` to generate an up-to-date localization file, and have committed both the `.res` file changes and the new localization `.txt` file
- [x] If I introduced new h/cpp files, I have added them to the appropriate project's VPC file (`server_momentum.vpc` / `client_momentum.vpc` / etc)
- [x] If I have added or modified any visual assets (models, materials, panels, effects, etc), I have taken screenshots / videos of them and attached them to this PR directly (screenshots uploaded through github, videos uploaded to youtube and linked)
- [x] If I have modified any console command, console variable, or momentum entity, I have opened an issue (or a PR) for it in the [Momentum Mod documentation repository](https://github.com/momentum-mod/docs)
- [x] My commits are relatively small and scoped to the best of my ability
- [x] My branch has a clear history of changes that can be easy to follow when being reviewed commit-by-commit
- [x] My branch is functionally complete; the only changes to be done will be those potentially requested in code review

### CODE REVIEW

**Review by braem — CHANGES_REQUESTED** [review]
Just realized something looking at this PR - I forgot to put the fizzle sound in the sound data of the RL & SL override scripts. Also the `chargestop` and `charged_shot` sound for the TF2 SL one. Could you add those while you're here?

I can make a separate PR if you'd prefer.

**Comment by Gocnak** [comment]
I'll do it here, should they just be the Momentum sounds?

**Comment by braem** [comment]
> 
> 
> I'll do it here, should they just be the Momentum sounds?

Actually to better emulate TF2 I suppose they shouldn't be added. However the TF2 stickybomb launcher override is using momentum sounds. This should be changed to
```
    SoundData
    {
        "single_shot"       "Weapon_StickyBombLauncher.Single"
        "charged_shot"      "Weapon_StickyBombLauncher.Single"
        "explosion"         "BaseExplosionEffect.SoundTF2"
        "detonate"          "Weapon_StickyBombLauncher.ModeSwitch"
        "deny"              "HL2Player.UseDeny"
        "charge"            "Weapon_StickyBombLauncher.ChargeUp"
    }
```

Though some of these overlap in channel, like the charge and det success (modeswitch) get cut off by others. Can be fixed by changing `"Weapon_StickyBombLauncher.ChargeUp"`'s channel to `CHAN_STATIC` and `"Weapon_StickyBombLauncher.ModeSwitch"`'s to `CHAN_AUTO`. Both are in `scripts/game_sounds_weapons_tf.txt`.

I suggest removing the momentum rocket trail sound from the RL ones as well, just so it's 1:1.

File: mp/game/momentum/custom/overrides/scripts/weapon_momentum_stickylauncher_tf2.txt [hunk 1]
```diff
@@ -26,11 +26,13 @@ WeaponData
...
+        "single_shot"       "Weapon_StickyBombLauncher.Single"
+        "charged_shot"      "Weapon_StickyBombLauncher.Single"
         "explosion"         "BaseExplosionEffect.SoundTF2"
-        "detonate"          "StickybombLauncher.Detonate"
-        "deny"              "StickybombLauncher.Deny"
-        "charge"            "StickybombLauncher.Charge"
+        "detonate"          "Weapon_StickyBombLauncher.ModeSwitch"
+        "deny"              "HL2Player.UseDeny"
+        "charge"            "Weapon_StickyBombLauncher.ChargeUp"
+        // "StickyFizzle"      "Stickybomb.Fizzle"
```
Comment (braem) [inline mp/game/momentum/custom/overrides/scripts/weapon_momentum_stickylauncher_tf2.txt:35]: One thing missed in the comments here is `"chargestop"`
```suggestion
        // "StickyFizzle"      "Stickybomb.Fizzle"
        // "chargestop"      "StickybombLauncher.ChargeStop"
```

**Review by braem — CHANGES_REQUESTED** [review]

**Review by braem — APPROVED** [review]

**Pull request merged**